web/views/pages/index.ejs

```diff
@@ -111,6 +111,11 @@ co2Sub = function(str) {
                         </span>
                     </div>
 
+                    <div class="pro-ad-header">
```

Comment:
Can you remove the `pro-` prefix in general? It's not really related to "pro" (which is not even one of our products anymore).
Also, did you make sure it's not shown on small devices? There's a CSS class you can add that does exactly that.

Reply:
Sure, I'll remove the pro prefix.

And yes (like I mentioned previously in this thread 😃), it is not shown on small devices. I'll update it to use the same visibility class though.